mlx_graphs/utils/transformations.py

```diff
@@ -433,22 +433,23 @@ def get_isolated_nodes_mask(
     """Returns a mask with isolated nodes set to ``False`` to filter them out if needed.
 
     Args:
-        edge_index : Edge index from which to remove isolated nodes.
-        num_nodes : Number of nodes of the graph.
-        complement : Wether to filter isolated or non isolated nodes.
+        edge_index: Edge index from which to remove isolated nodes.
+        num_nodes: Number of nodes of the graph.
+        complement: Wether to filter isolated or non isolated nodes.
 
     Returns:
         A boolean mask of size ``num_nodes`` where ``True`` means the node isn't
         isolated and ``False`` means it is.
 
-    Example :
-        >>> edge_index = mx.array([[0, 2, 0], [2, 0, 0]])
-        >>> mask = get_isolated_nodes_mask(edge_index, 3)
-        >>> mask
-        mx.array([0,2])
-        >>> mask = get_isolated_nodes_mask(edge_index, 3, complement=False)
-        >>> mask
-        mx.array([1])
+    Example:
+
+    .. code-block:: python
+
+        edge_index = mx.array([[0, 2, 0], [2, 0, 0]])
+        mask = get_isolated_nodes_mask(edge_index, 3)
+        # mx.array([0,2])
```

Comment:
Why replacing ´>>>´ by ´#´?

Reply:
I refactored it as a python block and messed up that part while doing it - I'll make it a `>>>`